fdm_requirements.txt

```diff
@@ -0,0 +1,8 @@
+pycodestyle
+mypy
+pytest
+pytest-cov
+pydbus
```

Comment:
What do you need pydbus for? It doesn't seem to be used here.

Reply:
That was a copy mistake